test/srdf_parser.test

```diff
@@ -1,3 +0,0 @@
-<launch>
```

Comment:
I understand that ros2 uses a different test approach, but is the replacement in this PR? I don't see it. If not, I think we should keep this with an in-line TODO (and maybe matching Issue).

Reply:
ros2 tests the .cpp without needing .test files